src/main/java/com/amplitude/api/DatabaseHelper.java

```diff
@@ -133,6 +139,7 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
             case 2:
                 db.execSQL(CREATE_IDENTIFYS_TABLE);
                 db.execSQL(CREATE_LONG_STORE_TABLE);
+                db.execSQL(CREATE_IDENTIFY_INTERCEPTOR_TABLE);
```

Comment:
Should we add a new DB version to add the new table?

Reply:
Add a new DB version and add unit tests, going to do an integration test